app/middlewares/albumfetcher.js

```diff
@@ -0,0 +1,23 @@
+const errors = require('../errors'),
+  config = require('../../config'),
+  logger = require('../logger'),
+  axios = require('axios'),
+  sessionsManager = require('../services/sessionsManager.js');
+
+exports.checkValidAlbumId = (req, res, next) => {
+  const url = `${config.common.albumList}/albums/${req.params.id}`;
+  return axios
```

Comment:
the responsibility of making the request to the external album provider should be encapsulated in some service and not directly in the middleware.